SwiftUI_DZ/SwiftUI_DZ/AutorizeView/AutorizeViewModel.swift

```diff
@@ -48,4 +68,26 @@ final class AutorizeViewModel: ObservableObject {
         guard charactersCount == Constants.loginTextFieldMaxLimitCharCount else { return false }
         return true
     }
+    
+    
+    func progressViewActivate() {
+        Timer.scheduledTimer(withTimeInterval: Constants.timeInterval, repeats: true, block: { timer in
+            if self.progressViewCount == Constants.progressMaxCount {
+                timer.invalidate()
+                if self.charactersCount < Constants.passwordMinLimitInt {
+                    withAnimation {
+                        self.sizeAnimation += Constants.oneNumber
+                        self.isShowIconError = true
+                        DispatchQueue.main.asyncAfter(deadline: .now() + Constants.oneDouble) {
+                            self.isShowIconError = false
+                        }
+                    }
+                } else {
+                    self.isDescriptionViewShown = true
+                }
+            } else {
+                self.progressViewCount += Constants.oneNumber
+            }
+        })
```

Comment:
```suggestion
            if self.progressViewCount == Constants.progressMaxCount {
                timer.invalidate()
                if self.charactersCount < Constants.passwordMinLimitInt {
                    withAnimation {
                        self.sizeAnimation += Constants.oneNumber
                        self.isShowIconError = true
                        DispatchQueue.main.asyncAfter(deadline: .now() + Constants.oneDouble) {
                            self.isShowIconError = false
                        }
                    }
                } else {
                    self.isDescriptionViewShown = true
                }
            } else {
                self.progressViewCount += Constants.oneNumber
            }
        })
```
больше ifов богу......